lstchain/reco/disp.py

```diff
@@ -36,7 +37,16 @@ def disp(cog_x, cog_y, src_x, src_y):
     """
     disp_dx = src_x - cog_x
     disp_dy = src_y - cog_y
-    disp_norm = np.sqrt(disp_dx**2 + disp_dy**2)
+    
+    disp_norm = disp_dx * np.cos(hillas_psi) + disp_dy * np.sin(hillas_psi)
+    disp_sign = np.sign(disp_norm)
+    disp_norm = abs(disp_norm)
```

Comment:
This should be `np.abs`

Reply:
> This should be `np.abs`

ok - to make it work also with arrays, right?